perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -57,6 +57,7 @@ BicycleTracker::BicycleTracker(
   float q_stddev_y = 0.6;                                     // [m/s]
   float q_stddev_yaw = tier4_autoware_utils::deg2rad(10);     // [rad/s]
   float q_stddev_vx = tier4_autoware_utils::kmph2mps(10);     // [m/(s*s)]
+  float q_stddev_ax = 9.8 / 2;                                // [m/(s*s)]
```

Comment:
Why is it divided by 2?

Reply:
I meant that the system noise of acceleration is half of the gravitational acceleration, but I have unified it with other descriptions.

Reply:
Why is it half the acceleration of gravity? :pray: